platform/files/order.go

```diff
@@ -0,0 +1,5 @@
+package files
+
+// order.toml is provided by the platform to enumerate the buildpacks and image extensions
```

Comment:
this file is great - is the implication that `buildpack.order` is the struct used to write "the platform's order.toml" ?   what does that mean re. the separation of the buildpack and platform apis?

Reply:
This is surely a point of intersection :)